creating_packages/package_information.rst

```diff
@@ -0,0 +1,81 @@
+
+    from conans import ConanFile
+
+    class MyConan(ConanFile):
+        name = "cool_library"
+        ...
+
+        def package_info(self):
+            self.cpp_info.includedirs = ["include/cool"]
+            self.cpp_info.libs = ["libcool"]
+            self.cpp_info.defines= ["DEFINE_COOL=1"]
+
+The package information is done using the attributes of the :ref:`cpp_info_attributes_reference` object. This information will be aggregated
+by Conan and exposed via ``self.deps_cpp_info`` to consumers and generators.
+
+Using Components
+----------------
+
+If your package contains more than one library or you want to define separated components so consumers can have more granular information,
+you can use components in your :ref:`method_package_info` method.
+
+.. warning::
+
+    This is a **experimental** feature subject to breaking changes in future releases.
+
+When you are creating a Conan package, it is recommended to have only one library (*.lib*, *.a*, *.so*, *.dll*...) per package. However,
+especially with third-party projects like Boost, Poco or OpenSSL, they would contain several libraries inside.
+
+Usually those libraries inside the same package depend on each other and modelling the relationship among them is required.
+
+With **components**, you can model libraries and executables inside the same package and how one depends on the other. Each library or
+executable will be one component inside ``cpp_info`` like this (the following case is not a real example):
+
+.. code-block:: python
+
+    def package_info(self):
+        self.cpp_info.name = "OpenSSL"
+        self.cpp_info.components["crypto"].names["cmake_find_package"] = "Crypto"
+        self.cpp_info.components["crypto"].libs = ["libcrypto"]
+        self.cpp_info.components["crypto"].defines = ["DEFINE_CRYPTO=1"]
+        self.cpp_info.components["ssl"].names["cmake"] = "SSL"
+        self.cpp_info.components["ssl"].includedirs = ["include/headers_ssl"]
+        self.cpp_info.components["ssl"].libs = ["libssl"]
+        self.cpp_info.components["ssl"].requires = ["crypto"]
+
+You can define dependencies among different components using the ``requires`` attribute and the name of the component. The dependency graph
+for components will be calculated and values will be aggregated in the correct order for each field.
+
+Declaration of requires from other packages is also allowed:
+
+.. code-block:: python
+
+    class MyConan(ConanFile):
+        ...
+        requires = "zlib/1.2.11", "openssl/1.1.1g"
+
+    def package_info(self):
+        self.cpp_info.components["comp1"].requires("zlib::zlib")             # Depends on all components in zlib package
+        self.cpp_info.components["comp2"].requires("comp1", "openssl::ssl")  # Depends on ssl component in openssl package
+
+.. important::
+
+    Note that component requires should **explicitly** define requirements to other package (``zlib::zlib``) or other package components
+    (``openssl::ssl``).
```

Comment:
I think this is important and it is not clear enough: _"By default components won't link against any other package required by the recipe, the `requires` list has to be populated explicitly with the list of components from other packages to use: it can be a single component (example) or the full requirement (example)."_